### PR TITLE
fixes csredis to use latest master commit and added contact us

### PR DESCRIPTION
**Pull request by computeronix**
--csredis is now using the latest from coinium/csredis
--added contact us page  https://github.com/bonesoul/CoiniumServ/issues/583

### CODE REVIEW

**Comment by bonesoul** [comment]
I guess the PR lacks ContactModule.cs

File: src/CoiniumServ/CoiniumServ.csproj [hunk 1]
```diff
@@ -171,6 +171,7 @@
     <Compile Include="Persistance\Layers\Hybrid\Migrations\M003FixDefaults.cs" />
     <Compile Include="Pools\IProfitInfo.cs" />
     <Compile Include="Pools\ProfitInfo.cs" />
+    <Compile Include="Server\Web\Modules\ContactModule.cs" />
```
Comment (bonesoul) [inline src/CoiniumServ/CoiniumServ.csproj]: The PR lacks the ContactModule.cs.

```
https://ci.appveyor.com/project/bonesoul/coiniumserv/build/1.0.775/job/3toy9y1rvv0k68fs
 CSC : error CS2001: Source file 'Server\Web\Modules\ContactModule.cs' could not be found [C:\projects\coiniumserv\src\CoiniumServ\CoiniumServ.csproj]
```

File: src/CoiniumServ/web/default/views/layout/sidebar.cshtml [hunk 2]
```diff
@@ -37,6 +37,7 @@
             <ul class="treeview-menu">
                 <li><a href="/api/" style="margin-left: 10px;"><i class="fa fa-signal"></i> API</a></li>
                 <li><a href="/tos/" style="margin-left: 10px;"><i class="fa fa-exclamation-circle"></i> Terms of Service</a></li>
+				<li><a href="/contact/" style="margin-left: 10px;"><i class="fa fa-question"></i> Contact Us</a></li>
```
Comment (bonesoul) [inline src/CoiniumServ/web/default/views/layout/sidebar.cshtml]: extra tabs here :)

Comment (bonesoul) [inline src/CoiniumServ/web/default/views/layout/sidebar.cshtml]: extra tabs here

**Comment by computeronix** [comment]
werid thought I added it :D  thanks for the tabs fixed  stupid windows tabs 🗡

File: src/CoiniumServ/CoiniumServ.csproj (hunk 1 above)
Comment (computeronix) [inline src/CoiniumServ/CoiniumServ.csproj]: fixed   didnt grab it from the first git command :(   it did now

**Pull request merged**